README.md

```diff
@@ -60,6 +60,7 @@ This project is not closed to actual static analyzers. With this repository we i
 ### C
 
 - [oclint](https://github.com/oclint/oclint) - Static analysis for C, C++ and Objective-C. Written in C++.
+- [uncrustify](https://github.com/uncrustify/uncrustify) - A source code beautifier for C, C++, C#, ObjectiveC, D, Java, Pawn and VALA
```

Comment:
I'm nitpicking here, but would you mind removing the 'A' at the beginning and adding a point at the end of the sentence? :)